src/components/Formik/Formik.stories.jsx

```diff
@@ -68,15 +100,55 @@ export const FormikForm = () => {
                 component={FormikTextInput}
               />
             </div>
-            <div style={{ marginBottom: '1rem' }}>
+            <div style={{ marginBottom: '1.25rem' }}>
               <Field
                 label="Email"
                 name="email"
                 id="email"
                 component={FormikTextInput}
+                isRequired
+              />
+            </div>
+            <div style={{ marginBottom: '1.25rem' }}>
+              <Field
+                label="Flavor"
+                name="flavor"
+                id="flavor"
+                options={flavorOptions}
+                component={FormikSelectInput}
+                // With a custom onChange.
+                // We preserve Formik's convention and relegate state management back to the user.
+                onChange={selectedOption => {
+                  action('change')(selectedOption);
+                  setFieldValue('flavor', selectedOption);
+                }}
+                isRequired
+              />
+            </div>
+            <div style={{ marginBottom: '1.25rem' }}>
+              <Field
+                label="Flavor no onChange"
+                name="flavor2"
+                id="flavor2"
+                options={flavorOptions}
+                component={FormikSelectInput}
+                isRequired
+              />
```

Comment:
@juanfabrega this Field does not provide an onChange function, but the Formik adapter doesn't wire up the setFieldValue automatically, so it doesn't work.